sparse-11/etc/ofono/ril_subscription.conf

```diff
@@ -0,0 +1,40 @@
+# Ofono configuration for Android 9
```

Comment:
Should we fix Android 9 or adjust comment to some modular?

Reply:
Will fix.